sammba/registration/struct.py

```diff
@@ -34,9 +35,35 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
         Maximal levels for each nonlinear warping iteration. Passed iteratively
         to sammba.externals.nipype.interfaces.afni.Qwarp
 
-    nonlinear_minimal_patch : int, optional
-        Minimal patch for the final nonlinear warp, passed to
+    nonlinear_minimal_patches : list of int, optional
+        Minimal patches for the final nonlinear warps, passed to
         sammba.externals.nipype.interfaces.afni.Qwarp
+        
+    nonlinear_weight : str, optional
+        Path to a mask used to weight non-linear registration.
```

Comment:
May be you can just add that this mask should contain brain and some scalp

Reply:
Done